module-types/HPE/Aruba-J9990A.yaml

```diff
@@ -0,0 +1,54 @@
+---
+manufacturer: HPE
+model: Aruba J9990A
+part_number: J9990A
+comments: HPE Aruba J9990A 20xGb PoE+/4x SFP+ V3 zl2 Module
+interfaces:
+  - name: '{module}1'
+    type: 1000base-t
```

Comment:
For Interfaces that provide POE+, please add the following lines:
`poe_mode: pse`
`poe_type: type2-ieee802.3at`